examples/jsm/tsl/math/Bayer.js

```diff
@@ -0,0 +1,18 @@
+import { TextureLoader } from 'three';
+import { Fn, int, ivec2, textureLoad } from 'three/tsl';
+
+let bayer16Texture = null;
+
+export const bayer16 = Fn( ( [ uv ] ) => {
```

Comment:
Currently writing the docs for this module...

What is this TSL function about? I've tried to find some references about "Bayer16" but without success.

Is this an alternative noise texture? I know the term "Bayer" only from Bayer Filters in context of image sensors but I don't understand how it relates to this code.

Reply:
Bayer Matrix is ​​a pattern, it is used for dithering effects as well, but in this case it was used as an alternative to blue-noise. I use `bayer16` which is a 16x16 texture with the bayer pattern, when used in Ray Marching specifically in `VolumeNodeMaterial.offsetNode`, it reduces `banding`, thus being able to use fewer `steps` without affecting the visuals as much.